pkg/udevevent/fdutil_test.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2018 The OpenEBS Author
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package udevevent
+
+import (
+	"syscall"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestFD_SET(t *testing.T) {
+	fds := [6]syscall.FdSet{
+		{[16]int64{1}},
+		{[16]int64{2}},
+		{[16]int64{4}},
+		{[16]int64{8}},
+		{[16]int64{16}},
+		{[16]int64{32}},
+	}
+	for index, fd := range fds {
+		tmpfd := &syscall.FdSet{}
+		FD_SET(tmpfd, index)
+		assert.Equal(t, fd, *tmpfd)
+	}
+}
+
+func TestFD_ISSET(t *testing.T) {
+	expectedOutput := [7]bool{true, true, true, true, true, true, true}
+	for index, output := range expectedOutput {
+		tmpfd := &syscall.FdSet{}
+		FD_SET(tmpfd, index)
+		assert.Equal(t, output, FD_ISSET(tmpfd, index))
+	}
+}
+
+func TestFD_ZERO(t *testing.T) {
```

Comment:
What is the intent of this test?

Reply:
If one fd is already set with some other value `FD_ZERO()` will reset that fd.